scripts/upgrade

```diff
@@ -55,16 +55,32 @@ chmod 750 "$install_dir"
 chmod -R o-rwx "$install_dir"
 chown -R ds:ds "$install_dir"
 
+#=================================================
+# ADVERTISE SERVICE IN ADMIN PANEL
+#=================================================
+
+for service in "ds-converter" "ds-docservice" "ds-metrics"; do
+    yunohost service add "$service"
+done
+
+#=================================================
+# START SERVICES
+#=================================================
+
+for service in "ds-converter" "ds-docservice" "ds-metrics"; do
+    ynh_systemd_action --action=restart --service_name="$service"
+done
+
 #=================================================
 # REAPPLY SYSTEM CONFIGURATIONS
 #=================================================
 ynh_script_progression --message="Upgrading system configurations related to $app..." --weight=1
 
-if [ $path = "/" ]; then
-    path_no_slash=""
-else
-    path_no_slash="$path"
-fi
+# if [ $path = "/" ]; then
+#     path_no_slash=""
+# else
+#     path_no_slash="$path"
+# fi
```

Comment:
I was thinking the same thing, but we need it here: https://github.com/YunoHost-Apps/onlyoffice_ynh/blob/604dcd8fd575b5d33b94cc306433b4bbba42db67/conf/nginx.conf#L9C1-L9C68

I'm not sure this line in nginx conf is useful though

Reply:
I have this line set to `proxy_set_header  X-Forwarded-Host $server_name;`

Reply:
which correspond to `$server_name__PATH_NO_SLASH__`, when path is `/` and since onlyoffice needs to be install in a root domain, let change it 👍